drivers/tty/serdev/pibridge.c

```diff
@@ -57,7 +57,6 @@ static int pibridge_parse_dt(struct serdev_device *serdev)
 
 	of_property_read_u32(node, "current-speed", &speed);
 	serdev_device_set_baudrate(serdev, speed);
-
```

Comment:
Actually I think this newline makes sense to separate the operations performed on the serial port, so I'd prefer if it could be kept.

Reply:
Ok